dev_info/build.html

```diff
@@ -0,0 +1,31 @@
+﻿<!DOCTYPE html>
+
+<link rel="stylesheet" href="../locales/help_enu/styles.css" type="text/css" />
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta charset="utf-8" />
+<title>Building GEDKeeper</title>
+</head>
+<body>
+<h1>How to build GEDKeeper</h1>
+<h2>Microsoft Windows</h2>
+<h3>Dependencies</h3>
+<p>The following is a list of required software to build and test:
+<ul>
+<li>Microsoft <a href="https://dotnet.microsoft.com/">.NET Framework</a> 4.0.30319</li>
+<li>Microsoft <a href="https://visualstudio.microsoft.com/">Visual Studio</a> 2017 or later</li>
+<li>Nullsoft Scriptable Install System (<a href="https://sourceforge.net/projects/nsis/">NSIS</a>) 3.04</li>
+<li><a href="https://sourceforge.net/projects/sevenzip/">7-zip</a></li>
+<li><a href="https://github.com/nunit">NUnit</a> 2.6.4 (unit testing framework for .NET)</li>
+</ul>
+</p>
+<h3>Execution</h3>
+<p>Run the executable <kbd>build_all_mswin.x86.cmd</kbd> from the command line.</p>
+<p>Optionally use the parameter &quot;Release&quot; for a release build (the default is a debug build). Testing is not done with a release build.</p>
```

Comment:
> release build

I'm not an English native speaker, and I may be wrong now, but I don't think \`\`release build'' is a good phrase for executable object(s) with debugging information stripped out.  The same for \`\`debug build''.  I understand that Microsoft named project configurations as "release" and "debug" to distinguish situations when output has and doesn't have debugging information.  And I also understand that \`\`release'' for a software product usually stands for "shipping to consumers" (and in very rare cases someone ships software with debugging information).  I just don't prefer to talk that way.

**May be** we should say something like this: "Pass \`\`Release'' option to the script to build GEDKeeper without debugging information (or *to build executable having no debugging information*, etc.); by default the script builds executables with embedded debugging information (*it's embedded in .NET executables, isn't it?*).  The script skips testing for non-debug builds.

I hope I could show you my general idea.  The text isn't a direction, though, it's just an idea.  Once again: I may be wrong with that.

Reply:
Good point. I have updated build.txt to address this. Though I believe HTML is a good way to have this file, I have only updated the text version because of @Serg-Norseman's concern.

Reply:
8790c4167d0d641b68b4355a79811e08a9a4400d is looked good to me.  Thanks for your work!